src/test/java/Phone.java

```diff
@@ -0,0 +1,20 @@
+public class Phone {
+    String number;
+
+    public Phone(String number) {
```

Comment:
As a phone can make calls for different numbers, I believe the `number` has to be a parameter to `call` method instead of the constructor.